### PR TITLE
Adding Spark 3.3/Image 2.1 acceptance tests

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

File: spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.12/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala212DataprocImage21AcceptanceTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
```
Comment (suryasoma) [inline spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.12/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala212DataprocImage21AcceptanceTest.java:2]: nit: 2022

**Review by suryasoma — APPROVED** [review]
LGTM

File: spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.12/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala212DataprocImage21AcceptanceTest.java (hunk 1 above)
Comment (davidrabinowitz) [reply spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.12/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala212DataprocImage21AcceptanceTest.java]: Thanks, fixed

**Pull request merged**